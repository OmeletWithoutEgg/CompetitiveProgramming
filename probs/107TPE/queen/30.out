-441202.04 -2847273.88
