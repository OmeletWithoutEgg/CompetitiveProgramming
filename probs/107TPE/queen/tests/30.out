377214.00 -35632384.00
